### PR TITLE
CQ-4305953 – Store theme as static files

### PR DESCRIPTION
**Pull request by basecode**
PR creates 

```
❯ tree
.
├── files
│   └── AEM_UI-kit_WKND.xd
├── previews
│   ├── articles.png
│   ├── carousel.png
│   ├── footer.png
│   └── teaser.png
├── properties.json
├── site-template.zip
├── site-theme.zip
└── theme-sources.zip
```

### CODE REVIEW

**Comment by bartoszglow** [comment]
As we have `theme-sources` and `site-template` as a `.zip` files maybe it would be worth it to have `theme-compiled` as a `.zip` as well?

**Comment by basecode** [comment]
@bartoszglow I thought about it and from a consistency POV it appears appealing but I decided against it for now because it would imply additional unzipping down the road which unnecessarily eats resources (memory + https traffic etc).

File: package.json [hunk 1]
```diff
@@ -4,26 +4,22 @@
   "description": "Generates an AEM Starter Kit",
   "license": "SEE LICENSE IN LICENSE.txt",
   "scripts": {
-    "build": "run-s build:theme build:clientlib build:template build:package",
+    "build": "run-s build:theme-compiled build:template build:package",
     "deploy": "curl -u admin:admin -F 'file=@starterkit.zip' http://localhost:4502/conf/global/starterkits.import.html",
-
-    "build:theme": "cd site.theme && npm install && npm run prod && cd ..",
-    "build:clientlib": "clientlib --verbose",
+    "build:theme-compiled": "cd site.theme && npm install && npm run prod && cd ..",
```
Comment (bartoszglow) [inline package.json]: I would stick to "build:theme" name. The result of building theme is the compiled theme but it doesn't have to be included in the name of npm script.

**Review by bartoszglow — COMMENTED** [review]
Please remove `aem-clientlib-generator` from `package.json` dev dependencies as it is no longer used

**Review by bartoszglow — APPROVED** [review]

**Pull request merged**